ra2ce/analyses/indirect/losses.py

```diff
@@ -180,7 +185,7 @@ def calculate_losses_from_table(self):
         """
 
         traffic_data = self.load_df(
-            self.config["input"] / "losses", "traffic_intensities.csv"
+            self.config.input_path / "losses", "traffic_intensities.csv"
```

Comment:
you might as well create in the `__init__` the property `self._losses_input_dir = self.config.input_path.join("losses")` as it's accessed three times already.

Reply:
Made a local var as all 3 occurrences are in the same function.